var/spack/repos/builtin/packages/py-neurodamus/package.py

```diff
@@ -38,6 +38,7 @@ class PyNeurodamus(PythonPackage):
     depends_on('py-docopt',        type='run')
     depends_on('py-lazy-property', type='run', when='@:1.0.0')
     depends_on('py-six',           type='run', when='@:1.0.0')
+    depends_on('py-morphio+mpi',   type='run', when='@develop')
```

Comment:
Stupid question, but why does `py-morphio` need a `+mpi` variant? Just for the concretizer?